docs/DeveloperGuide.md

```diff
@@ -257,27 +257,34 @@ _{Explain here how the data archiving feature will be implemented}_
 
 **Target user profile**:
 
+* is a university student who organizes many bulk purchases
 * has a need to manage a significant number of contacts
+* has a need to track who has paid him back
 * prefer desktop apps over other types
 * can type fast
 * prefers typing to mouse interactions
 * is reasonably comfortable using CLI apps
 
-**Value proposition**: manage contacts faster than a typical mouse/GUI driven app
+**Value proposition**:  
+1. track their debtors
+2. view total money owed from all debtors
+3. manage contacts faster than a typical mouse/GUI driven app
```

Comment:
Apparently the markdown guide says should use generic numbering for ordered lists i.e. all items start with 1.